lib/scope.js

```diff
@@ -909,36 +921,73 @@ module.exports = {
+
+      if ( !field.trigger ) {
+        // field/page trigger var is needed but is missing.
+        let page_trigger_warning = `A field on this page uses an index `
+          + `variable or a generic object, but the interview page does not `
+          + `give the trigger variable's name. See `
+          + `https://suffolklitlab.org/docassemble-AssemblyLine-documentation`
+          + `/docs/automated_integrated_testing/#a-missing-trigger-variable.`;
+        await scope.addToReport(scope, { type: `warning`, value: page_trigger_warning });
+      }
 
       let original_matches = matches;
+      let fallbacks = [];
       matches = [];
       for ( let match of original_matches ) {
-        // Feedback for devs - we need the trigger var but we don't have it (var data)
-        if ( match.trigger === undefined ) { await scope.addToReport(scope, { type: `warning`, value: `A field on this page uses an index variable or a generic object. The test gave story data for setting the field, but that data is missing the trigger. Add a trigger variable to the story table \`trigger\` column. For now, the test may end up ignoring this field. The row's data is\n${ JSON.stringify( match.original )}.\nThe field's data is\n${ JSON.stringify( field )}` }); }
-          // TODO: Add a warning if they're both `undefined`?
-        else {
-
+        if ( !match.trigger ) {
+          // var/row trigger data is (probably) needed but is missing. Caveat: If there's
+          // only one page in the proxy var loop and there's only one row, trigger var is
+          // not needed, but I'm not sure how we figure out all those things here and log
+          // a useful message.
+          let row_trigger_warning = `This story table row needs the "trigger" `
+            + `value, but it's missing. The test might get unexpected results. See `
+            + `https://suffolklitlab.org/docassemble-AssemblyLine-documentation`
+            + `/docs/automated_integrated_testing/#trigger.`
+            + `\nThe row's data is\n${ JSON.stringify( match.original )}.`
+            + `\nThis field's data is\n${ JSON.stringify( field )}`;
+          await scope.addToReport(scope, { type: `warning`, value: row_trigger_warning });
+          fallbacks.push( match );
+        } else {
           let triggers_match = await scope.matchNeutralizedQuotes( scope, { value1: field.trigger, value2: match.trigger });
           if ( triggers_match ) {
             matches.push( match );
           }
         }
+      }  // ends for each match
+
+      // Be very permissive, based on feedback. See #464 comment.
+      // Use undefined trigger row(s) if no matching triggers were found
+      if ( matches.length === 0 ) { matches = fallbacks; }
+
+      // Feedback for devs on multiple row match without any matching triggers
+      if ( original_matches.length > 1 && matches.length === 0 ) {
+        let missing_row_warning = `The story table has multiple rows `
```

Comment:
We have decided to put "Is the story table missing a row?" as the first sentence.